### PR TITLE
Add pyboost_ipc, pytest-incremental and update doit to 0.29 and pytest to 2.8.4

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: ipython/bld.bat [hunk 1]
```diff
@@ -5,4 +5,4 @@ python setup.py install
 if errorlevel 1 exit 1
 
 rd /s /q %SCRIPTS%
-rd /s /q %SP_DIR%\ipython-%PKG_VERSION%-py%PY_VER%.egg\share
```
Comment (ccordoba12) [inline ipython/bld.bat]: Please leave this line.

Comment (nicoddemus) [inline ipython/bld.bat]: If I leave that line, I get this error on Windows:

```
rd /s /q E:\Miniconda\envs\_build\Lib\site-packages\ipython-3.2.1-py2.7.egg\share
The system cannot find the path specified.
Command failed: C:\Windows\system32\cmd.exe /c call bld.bat
```

Comment (ccordoba12) [inline ipython/bld.bat]: Ok, what if you remove just the `share` dir at the end and try again?

Comment (nicoddemus) [inline ipython/bld.bat]: Same error:

```
 rd /s /q E:\Miniconda\envs\_build\Lib\site-packages\ipython-3.2.1-py2.7.egg
The system cannot find the file specified.
Command failed: C:\Windows\system32\cmd.exe /c call bld.bat
```

Comment (ccordoba12) [inline ipython/bld.bat]: Ok, then please remove it :-) Sorry for the noise.

Comment (nicoddemus) [inline ipython/bld.bat]: No worries. If there's nothing else this can be merged then.

File: cquadpack/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package:
+  name: cquadpack
+  version: 1.0
+
+source:
+  git_url: git@github.com:ESSS/cquadpack.git
```
Comment (ccordoba12) [inline cquadpack/meta.yaml]: Isn't `git_tag` or `git_rev` missing here?

File: cquadpack/bld.bat [hunk 3]
```diff
@@ -0,0 +1,15 @@
+if "%CMAKE_GENERATOR%" == "" (
```
Comment (ccordoba12) [inline cquadpack/bld.bat]: Please set `CMAKE_GENERATOR` to `NMake Makefiles`. That will detect and compile this program for the right VS version.

File: pyboost_ipc/bld.bat [hunk 4]
```diff
@@ -0,0 +1,16 @@
+if "%CMAKE_GENERATOR%" == "" (
```
Comment (ccordoba12) [inline pyboost_ipc/bld.bat]: The same here about `NMake Makefiles`

File: cquadpack/meta.yaml (hunk 2 above)
Comment (nicoddemus) [inline cquadpack/meta.yaml]: Indeed. Fixed!

File: pyboost_ipc/bld.bat (hunk 4 above)
Comment (nicoddemus) [inline pyboost_ipc/bld.bat]: Done

File: cquadpack/bld.bat (hunk 3 above)
Comment (nicoddemus) [inline cquadpack/bld.bat]: True, done!

File: pyboost_ipc/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+package:
+  name: pyboost_ipc
+  version: 0.1
+
+source:
+  git_url: git@github.com:ESSS/pyboost_ipc.git
```
Comment (ccordoba12) [inline pyboost_ipc/meta.yaml]: `git_tag` is also missing here

File: cquadpack/bld.bat (hunk 3 above)
Comment (ccordoba12) [inline cquadpack/bld.bat]: I don't like this too much. Why not make it:

    set CMAKE_GENERATOR=NMake Makefiles

?

I mean, I don't think there's anything to be gained by letting users to override CMAKE_GENERATOR from the environment.

File: pyboost_ipc/meta.yaml (hunk 5 above)
Comment (nicoddemus) [inline pyboost_ipc/meta.yaml]: Fixed.

**Comment by nicoddemus** [comment]
Always setting `CMAKE_GENERATOR` for `cquadpack` and `pyboost_ipc` as requested.

**Comment by ccordoba12** [comment]
Ok, everything seems good now. Thanks for your patience :-)

**Comment by nicoddemus** [comment]
No worries, thanks!

**Pull request merged**